fn_guardium_insights_integration/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# docker build -t resilient/{ext} .
```

Comment:
We've made a few changes to the dockerfile template. Take a look at https://github.ibm.com/Resilient/resilient-community-apps/blob/master/fn_outbound_email/Dockerfile. In particular, lines 44 and 49 for the new group

Reply:
resolved